docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -35,18 +41,27 @@ while the other pods shouldn't tolerate the taint. The taint is removed once the
 
 *Warning:* currently there is no guarantee which node is chosen and which pods are being killed
 in order to schedule critical pods, so if rescheduler is enabled your pods might be occasionally
-killed for this purpose.
+killed for this purpose. Please ensure that rescheduler is enabled along with priorities & preemptions in default-scheduler as it may have unwanted side-effects.
 
 ## Config
 
-Rescheduler should be enabled by default. It doesn't have any user facing configuration (component config) or API.
-Rescheduler may also be disabled.
+Rescheduler doesn't have any user facing configuration (component config) or API.
 
-### Marking add-on as critical
+### Marking pod as critical when using Rescheduler(This will be deprecated).
 
-To be critical an add-on has to run in `kube-system` namespace (configurable via flag) and
+To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
```

Comment:
> the pod

an add-on pod

Reply:
So this is for any pod, not only add-on pods.